taxcalc/functions.py

```diff
@@ -915,7 +954,7 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517,
     _alminc = c62700
 
     if (c02700 > 0):
-        _alminc = max(0., c62100 - c62600 + c02700)
+        _alminc = max(0., c62100 - c62600)
```

Comment:
Previously we have `_alminc = c62700 = max(0., c62100 - c62600)`, so this clause here is merely duplicates and does not make a lot of sense to me, especially along with the prior condition `(c02700 > 0)`.